pkg/bootstrapper/bootstrapper.go

```diff
@@ -182,9 +182,12 @@ func NewWinNodeBootstrapper(k8sInstallDir, ignitionFile, kubeletPath string, cni
 	var dependents []*mgr.Service
```

Comment:
I think the approach taken here requires a commit/PR title change, something like `Update kubelet service in place instead of recreating`

Reply:
Given this is a bug, I am not sure if the title should be same as in Bugzilla

Reply:
I missed this. Please don't repeat the title of the bug in the PR. That is akin to saying you are introducing the bug. Mention **how** you are fixing the bug.